packages/hardware-trezor/src/TrezorKeyAgent.ts

```diff
@@ -16,9 +16,12 @@ import {
   util
 } from '@cardano-sdk/key-management';
 import { txToTrezor } from './transformers/tx';
-import TrezorConnectWeb from '@trezor/connect-web';
+import _TrezorConnectWeb from '@trezor/connect-web';
 
 const TrezorConnectNode = Trezor.default;
+const TrezorConnectWeb = (_TrezorConnectWeb as any).default
+  ? ((_TrezorConnectWeb as any).default as typeof _TrezorConnectWeb)
+  : _TrezorConnectWeb;
```

Comment:
@mkazlauskas This won't work in the Lace :( 
I made the same change when I was debugging it in the scope of https://input-output.atlassian.net/wiki/spaces/CWT/pages/4129554601/External+Dapps+testing+Trezor ticket.
It looks good when you are testing it in SDK but you will still have `.default` when the method is called from Lace

Reply:
Update: 
I checked this comment https://input-output-rnd.slack.com/archives/C01S2DD5B0D/p1708699319597529?thread_ts=1708695474.577899&cid=C01S2DD5B0D

Maybe that wasn't working because I tested it by using / setting `dist` from SDK directly. Let me check it.

Reply:
@mkazlauskas Checked, problem was in the Lace webpack. 
Ignore previous comments!